### ISSUE TITLE
[Feature Request] Option to decrease vibrance of album art auto-theming

### ISSUE THREAD

**Issue by zkldi** [Taiko2k/TauonMusicBox#564]
At the moment I'm using the auto-theme option because it looks really cool, but for certain albums it can cause some rather aggressive changes in theme colours. Could there be a way to 'merge' the album art colour as a kind of highlight colour with an existing theme, or just a way to tone down the colours in general?

As an example of the current setting, going from album to album results in some bleedingly bright colours.
![Peek 2021-09-12 18-12](https://user-images.githubusercontent.com/20380519/132996620-c492c375-3b8b-45ac-9477-f32c43a6c202.gif)

**Comment by zkldi** [comment]
Also, I'd be willing to make a PR for this if that's cool (and it wouldnt be too hard :upside_down_face:)

**Comment by Taiko2k** [comment]
> Could there be a way to 'merge' the album art colour as a kind of highlight colour with an existing theme

I think that a themes design would need to be intended for this, not just applied to any theme.

The built-in theme "carbon" actually already does this. (This is a much different, controled process than the auto-thene function)

> Option to decrease vibrance of album art auto-theming

Sometimes this feature produces good results, sometimes bad with harsh contrast/vibrance such as in your example. While the whole approach could be changed and added as a sub option, I don't wanna go adding such; what I would call "bandaid options". Ideally it would just always produce good results. 

Better solution would be to detect when the existing algorithm has produced a harsh result (Easy enough), then have an alternative process to use in that case (Kind of hard as im not sure what that could be. I don't think simply lowing the vibrance would work)

I actually sometimes think of just removing auto-theme.

**Comment by zkldi** [comment]
Yeah, I think you're right. I had no idea the carbon theme would read the album art for highlight colours, and it looks great imo.

I think the auto-theme option could be removed then, and some sort of tag could be added to themes that read the album art?

That sounds cool, anyway. Thanks for such an immediate response!
